src/ui/utils/devtools-toolbox.js

```diff
@@ -37,6 +39,13 @@ export class DevToolsToolbox {
     await this.selectTool(selectedPanel);
   }
 
+  initConsole() {
+    initOutputSyntaxHighlighting();
+
+    // TODO: the store should be associated with the toolbox
+    setupMessages(store);
```

Comment:
Is there a reason why we can't just move `setupMessages` to the end of main.js? 

https://github.com/RecordReplay/devtools/blob/master/src/main.js#L81